androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/components/TitledSectionTemp.kt

```diff
@@ -0,0 +1,57 @@
+package com.EENX15_22_17.digital_journal.android.ui.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.EENX15_22_17.digital_journal.android.ui.theme.colorTextGray
+
+@Composable
+fun TitledSectionTemp(title: String, body: @Composable() () -> Unit) {
+    Box(
+        Modifier
+            .fillMaxWidth()
+            .padding(bottom = 10.dp, top = 10.dp)) {
+        Box(
+            Modifier
```

Comment:
I think we can keep modifiers on the same line as the Box when we have few chained modifiers, as this makes the indentation tree cleaner.